pkg/lib/facade/coordinator.go

```diff
@@ -236,6 +236,15 @@ func (c *Coordinator) IdentityCreateByAdmin(userID string, spec *identity.Spec,
 		return nil, err
 	}
 
+	// Remove any anonymous identity this user has.
+	if err := c.removeAnonymousIdentitiesOfUser(userID); err != nil {
+		return nil, err
+	}
+
+	if err := c.StdAttrsService.PopulateIdentityAwareStandardAttributes(userID); err != nil {
```

Comment:
It seems it is a bit easy for us to call this function after updated any identities.